patches/minecraft/net/minecraft/entity/monster/AbstractSkeletonEntity.java.patch

```diff
@@ -1,6 +1,26 @@
 --- a/net/minecraft/entity/monster/AbstractSkeletonEntity.java
 +++ b/net/minecraft/entity/monster/AbstractSkeletonEntity.java
-@@ -155,7 +155,7 @@
+@@ -45,6 +45,8 @@
+ import net.minecraft.world.DifficultyInstance;
+ import net.minecraft.world.IServerWorld;
+ import net.minecraft.world.World;
++import net.minecraftforge.common.MinecraftForge;
++import net.minecraftforge.event.entity.living.LivingBurnBySunlightEvent;
+ 
+ public abstract class AbstractSkeletonEntity extends MonsterEntity implements IRangedAttackMob {
+    private final RangedBowAttackGoal<AbstractSkeletonEntity> field_85037_d = new RangedBowAttackGoal<>(this, 1.0D, 20, 15.0F);
+@@ -109,7 +111,9 @@
+          }
+ 
+          if (flag) {
+-            this.func_70015_d(8);
++            LivingBurnBySunlightEvent burnEvent = new LivingBurnBySunlightEvent(this, 8);
++            MinecraftForge.EVENT_BUS.post(burnEvent);
++            this.func_70015_d(burnEvent.getDuration());
```

Comment:
Make this a method in ForgeEventFactory instead, to have a one line patch instead of 3 each time.